### PR TITLE
[v7r0] Hackathon fixes

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*WMS
CHANGE: reverted JobsStatus table

*tests
CHANGE: distinguish client and server tests

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/Service/JobMonitoringHandler.py [hunk 1]
```diff
@@ -438,8 +435,10 @@ def export_getJobPageSummaryWeb(self, selectDict, sortList, startItem, maxItems,
           hbTime = Time.fromString(jobDict['HeartBeatTime'])
           # There is no way to express a timedelta of 0 ;-)
```
Comment (andresailer) [inline WorkloadManagementSystem/Service/JobMonitoringHandler.py]: Apparently this comment is (now?) wrong?

**Comment by fstagni** [comment]
Please review.

File: Core/scripts/dirac-install.py [hunk 2]
```diff
@@ -2319,8 +2319,9 @@ def createBashrcForDiracOS():
       lines = ['# DIRAC bashrc file, used by service and agent run scripts to set environment',
                'export PYTHONUNBUFFERED=yes',
                'export PYTHONOPTIMIZE=x',
-               '[ -z "$DIRACOS" ] && export DIRACOS=%s/diracos' % proPath,
-               '. %s/diracos/diracosrc' % proPath]
+               '[ -z "$DIRAC" ] && export DIRAC=%s' % proPath,
+               '[ -z "$DIRACOS" ] && export DIRACOS=$DIRAC/diracos',
+               '. $DIRAC/diracos/diracosrc']
```
Comment (andresailer) [inline Core/scripts/dirac-install.py]: There was some discussions on slack that one might want to set DIRACOS to some other path before sourcing the bashrc (if I understand correctly?) In that case shouldn't this line be `'. $DIRACOS/diracosrc']` ?

Comment (fstagni) [reply Core/scripts/dirac-install.py]: Yes it's better. Fixed.

**Comment by atsareg** [comment]
There are conflicts in this branch, please rebase it.

**Comment by atsareg** [comment]
The conflict was trivial, I have resolved it online

**Pull request merged**